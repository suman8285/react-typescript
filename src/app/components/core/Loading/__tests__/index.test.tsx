import React from 'react';
import { render } from '@testing-library/react';
import { Loading } from '../index';

describe('Loading Component', (): void => {
  it('Matching snapshot', () => {
    const { asFragment } = render(<Loading />);
    expect(asFragment()).toMatchSnapshot();
  });
});
