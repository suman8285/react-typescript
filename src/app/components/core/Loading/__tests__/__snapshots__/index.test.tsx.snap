// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component Matching snapshot 1`] = `
<DocumentFragment>
  .c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  position: absolute;
  left: 0;
  right: 0;
  top: 0;
  bottom: 0;
  background: none;
  z-index: 29999;
}

<div
    class="c0"
  >
    <div
      aria-label="animation"
      role="button"
      style="width: 100px; height: 100px; overflow: hidden; margin: 0px auto; outline: none;"
      tabindex="0"
      title=""
    >
      <svg
        height="400"
        preserveAspectRatio="xMidYMid meet"
        style="width: 100%; height: 100%; transform: translate3d(0,0,0);"
        viewBox="0 0 400 400"
        width="400"
        xmlns="http://www.w3.org/2000/svg"
      >
        <defs>
          <clippath
            id="__lottie_element_2"
          >
            <rect
              height="400"
              width="400"
              x="0"
              y="0"
            />
          </clippath>
        </defs>
        <g
          clip-path="url(#__lottie_element_2)"
        >
          <g
            opacity="1"
            style="display: block;"
            transform="matrix(0.7741100192070007,0,0,0.7741100192070007,200,205)"
          >
            <g
              opacity="1"
              transform="matrix(0.7222599983215332,0,0,0.7222599983215332,2.5490000247955322,-5.7170000076293945)"
            >
              <path
                d=" M-54.11600112915039,-133.9810028076172 C-37.39899826049805,-140.7480010986328 -19.131000518798828,-144.47500610351562 0,-144.47500610351562 C0,-144.47500610351562 0,-144.47500610351562 0,-144.47500610351562 C21.639999389648438,-144.47500610351562 42.17499923706055,-139.70599365234375 60.611000061035156,-131.16400146484375"
                fill-opacity="0"
                stroke="rgb(0,175,239)"
                stroke-linecap="round"
                stroke-linejoin="miter"
                stroke-miterlimit="4"
                stroke-opacity="1"
                stroke-width="19"
              />
            </g>
            <g
              opacity="1"
              transform="matrix(1.0813900232315063,0,0,1.0813900232315063,2.5490000247955322,-5.7170000076293945)"
            >
              <path
                d=" M-135.7740020751953,-49.45600128173828 C-129.47500610351562,-66.72200012207031 -119.9739990234375,-82.4540023803711 -107.99199676513672,-95.93099975585938"
                fill-opacity="0"
                stroke="rgb(0,175,239)"
                stroke-linecap="round"
                stroke-linejoin="miter"
                stroke-miterlimit="4"
                stroke-opacity="1"
                stroke-width="14"
              />
            </g>
          </g>
        </g>
      </svg>
    </div>
  </div>
</DocumentFragment>
`;
